### PR TITLE
update(CI): Publish wasm package as wasm-dev

### PR DESCRIPTION
**Pull request by Rohith-Raju**
**What type of PR is this?**

/kind feature

**Any specific area of the project related to this PR?**

/area CI

**What this PR does / why we need it**: 
This `PR` will help add a new feature to [falco-playground ](https://github.com/falcosecurity/falco-playground) where a user can select multiple versions of `falco`.  Since we don't support `wasm` "officially" it is a `dev` only package for now.  

**Which issue(s) this PR fixes**:  https://github.com/falcosecurity/falco-playground/issues/13

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
update(CI): publish wasm package as dev-wasm 
```

### CODE REVIEW

**Comment by leogr** [comment]
/assign @LucaGuerra 

as discussed :pray:

File: scripts/publish-wasm [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+          ;;
+    esac
+done
+shift $((OPTIND-1))
+
+if [ -z "${file}" ]; then
+    usage
+fi
+
+repo="dev-wasm"
```
Comment (LucaGuerra) [inline scripts/publish-wasm:27]: `wasm-dev` for consistency (sorry, I also always get that mixed up): https://download.falco.org/?prefix=packages/

**Comment by Rohith-Raju** [comment]
> PR is still in draft, do you think there is anything missing?

Made it a draft to check my approach and permissions, I'll disable it now 

>Also just to double check, WASM does not have any other architecture right, it's like an architecture of its own with its own format? In that case I think the directory is right.

Correct!! Wasm is platform-independent. 

>Also permissions should already be OK on the S3 bucket to publish this.

Great!!

**Review by FedeDP — APPROVED** [review]
/approve

**Pull request merged**